documentation/API proposals/APIproposal_QualityByDesign_CableLabs.md

```diff
@@ -0,0 +1,16 @@
+| YAML code available?                      | YES<br>- yaml openapi spec<br>- yaml - Swagger Editor<br>- slides                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| Validated in lab/productive environments? | YES - app using APIs validated in CableLabs lab with potential field trials coming soon with network operators.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| Validated with real customers?            | NO - not yet, but CableLabs is currently in talks with both Meta and Zoom. Use case for both are for video conferencing applications to be able to monitor video, send KPIs, and trigger events if thresholds drop below certain levels.                                                                                                                                                                                                                                                                                                                                                            |
+| Validated with operators?                 | NO - but actively working with Charter, Liberly Global, and Vodaphone to get up and running in their labs.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
Confirm that you have the blessing of these vendors and operators before you share their name.  If we don't have it yet, I'd in progress.

Reply:
updated to remove external customers but left vendor names in